packages/core/admin/ee/server/services/audit-logs.js

```diff
@@ -71,6 +75,11 @@ const createAuditLogsService = (strapi) => {
       return null;
     }
 
+    // Ignore events without a user
+    if (!strapi.requestContext.get()?.state?.user) {
```

Comment:
`ctx.state.user` can also represent an [authenticated U&P user](https://github.com/strapi/strapi/blob/main/packages/plugins/users-permissions/server/strategies/users-permissions.js#L51)
Do we want to also log events from those users?

Reply:
```suggestion
    const { isNil, propEq } = require('lodash/fp');

    const state = strapi.requestContext.get()?.state ?? {};

    const isUsingAdminAuthStrategy = propEq('strategy.name', 'admin', state);
    const hasAuthenticatedUser = !isNil(state.user);

    // Ignore events without an authenticated user using the admin auth strategy
    if (!hasAuthenticatedUser || !isUsingAdminAuthStrategy) {
      return;
    }
```